File: applications/summersim18/wrapped/python/div-showcase/left-div.py
from numb import *

if __name__ == '__main__':
    Numb.setup(cache_out='div-cache1', precision=20, strategy='ignore-cache')
    a = Numb(100)
    b = Numb(1e308)
    c = Numb(11)
    
    d = (a / b) / c
    print "(100 / 1e308) / 11 = {0}".format(d)

File: applications/summersim18/wrapped/cpp/div-showcase/left-div.cpp
#include <stdio.h>
#include <stdlib.h>
#include <math.h>
#include "../../../../../cpp/numb/numb.hpp"

int main()
{
    Numb::setup("div-cache1", "", 20, "ignore-cache");
    Numb a(100);
    Numb b(1e308);
    Numb c(11);

    Numb d = (a / b) / c;
    cout << "(100 / 1e308) / 11 = " << d << endl;

    return 0;
}
